util/coinach.ts

```diff
@@ -154,6 +154,23 @@ export class CoinachWriter {
     return '.';
   }
 
+  sortObjByKeys(_obj: unknown): unknown {
+    const obj = _obj as Record<string, unknown>;
```

Comment:
Is it possible to do this without type assertion? `sortObjByKeys` could be called with a `number` or something that you can't call Obect.keys on.

I don't think there's a good way to go from unknown to an indexable type.  (You can get to `object` or `{}` but there's lint banning those.)  You could add a type guard though, which seems slightly better than a type assertion.

```typescript
const isObject = (obj: unknown): obj is { [key: string]: unknown } => {
  return obj !== null && typeof obj === 'object' && !Array.isArray(obj);
};

  sortObjByKeys(obj: unknown): unknown {
    if (!isObject(obj))
      return;
    const out = Object
      .keys((obj))
      .sort()
      .reduce((acc: typeof obj, key) => {
        const nested = obj[key];
        if (isObject(nested))
          acc[key] = this.sortObjByKeys(nested);
        else
          acc[key] = nested;

        return acc;
      }, {});
    return out;
  }
```

Reply:
Done.
I have no better solution either...